Stratis.SmartContracts.Standards/Stratis.SmartContracts.Standards.csproj

```diff
@@ -1,12 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <Version>2.0.0</Version>
+    <TargetFramework>net6.0</TargetFramework>
+    <Version>1.0.0</Version>
```

Comment:
I am expecting to see `2.0.1` - same as the `Stratis.SmartContracts` version. Is this correct?

Reply:
Yes